src/app/store/effects/app-form.effects.spec.ts

```diff
@@ -29,7 +30,7 @@ describe('AppFormEffects', () => {
     expect(effects).toBeTruthy();
   });
 
-  describe('setAppFormPristine$', () => {
+  describe('setTechFormPristine$', () => {
     it('should dispatch SetAppFormPristine action if view state is set to view only', () => {
```

Comment:
use `should dispatch SetAppFormPristine & ClearErrorMessage actions if view state is set to view only`